src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -182,7 +169,8 @@ export const SubmissionHistory = () => {
                   field: 'duration',
                   headerRenderer: () => h(Sortable, { sort, field: 'duration', onSort: setSort }, ['Duration']),
                   cellRenderer: ({ rowIndex }) => {
-                    return h(TextCell, [Utils.customFormatDuration(runSetDuration(paginatedPreviousRunSets[rowIndex]))])
+                    const row = paginatedPreviousRunSets[rowIndex]
+                    return h(TextCell, [Utils.customFormatDuration(getDuration(row.state, row.submission_timestamp, row.last_modified_timestamp, isRunSetInTerminalState))])
```

Comment:
How come this one needs a custom duration format vs the others locations?

Reply:
So whenever we want to display duration we use custom format to make a human readable format as compared to [this line](https://github.com/DataBiosphere/cbas-ui/blob/86f4d6d3d5d939e11a85a8f092b27a62e02f2de9/src/pages/SubmissionHistory/SubmissionHistory.js#L30) where we just calculate duration per run set and save it back to run set data.